Suggested changes to ref arch nav and description (#17083)
cloudflare/cloudflare-docs: src/content/docs/reference-architecture/how-to-use.mdx

---
title: How to use
pcx_content_type: overview
sidebar:
  order: 2
---

import { CardGrid, Description, LinkTitleCard, Render } from "~/components";

<Description>
	The reference architecture content in our documentation is designed to help you understand how Cloudflare has been designed and built,
	and how our products and services integrate with your current IT architecture.

</Description>

Below are the different types of architecture content and information is organized from high level reference architectures, to design guides with best practices and guidelines to implementation guides which provide detailed steps to deploy a specific solution.

## Reference architectures

> [Reference architecture](/reference-architecture/architectures/) provide a foundational knowledge of the Cloudflare platform and products while offering a description for how they relate to your existing infrastructure and business challenges. They are high-level, conceptual documents that walk through the concepts of an area of our platform, mapping our network, products and features to the typical architecture of a customer's environment. Detailed diagrams with supporting content explain how our technology works and how it can be integrated with your own infrastructure. The goal of a reference architecture is:

Reviewer: ```suggestion
[Reference architectures](/reference-architecture/architectures/) provide a foundational knowledge of the Cloudflare platform and products while offering a description for how they relate to your existing infrastructure and business challenges. They are high-level, conceptual documents that walk through the concepts of an area of our platform, mapping our network, products and features to the typical architecture of a customer's environment. Detailed diagrams with supporting content explain how our technology works and how it can be integrated with your own infrastructure. The goal of a reference architecture is:
```

## Design guides

These [guides](/reference-architecture/design-guides/) are typically aimed at architects, developers, and IT professionals who are tasked with designing and deploying systems that leverage the company's technologies. They typically focus on a specific solution that would be a subset of the greater architecture. For example, if you have read our [SASE Reference Architecture](/reference-architecture/architectures/sase/), but are a startup, you may want to understand the details of using a [SASE approach for a small startup](/reference-architecture/design-guides/zero-trust-for-startups/). These documents are:

- Helping you think through how to design a deployment of Cloudflare as part of an overall solution.
- More prescriptive than reference architectures, sharing best practices and guidelines.
- Focused on a solution design that you are trying to achieve, such as connecting private networks to Cloudflare, or using a web application firewall to secure a public website.
- Not a replacement for product documentation and do not describe specific product configuration or commands to run.

## Implementation guides

> Detailed documents that provide [step-by-step instructions](/reference-architecture/implementation-guides/) and practical guidance for how to effectively deploy and configure specific solutions or services and are typically focused on a specific implementation goal. A design guide provides the overall best practices for designing a solution, while the implementation guide details the actual steps to deploy. These documents are:

Reviewer: ```suggestion
Implementation guides provide [step-by-step instructions](/reference-architecture/implementation-guides/) and practical guidance for how to effectively deploy and configure specific solutions or services. Implementation guides are focused on a specific implementation goal. While a design guide provides the overall best practices for designing a solution, an implementation guide details the actual steps to deploy in the context of a specific job-to-be-done. These documents are:
```

> - Focused on a specific implementation outcome, that is, connecting a remote office using the Magic WAN Connector.

Reviewer: ```suggestion
- Focused on a specific implementation outcome, such as connecting a remote office using the Magic WAN Connector.
```

> - Information the exact commands and configuration steps to take.

Reviewer: ```suggestion
- Provide information about the exact commands and configuration steps to take.
```
